Dockerfile-expocli

```diff
@@ -1,4 +1,6 @@
 FROM countingup/node:12
 
+LABEL org.opencontainers.image.source="https://github.com/Countingup/docker-node"
```

Comment:
I think this is correct, even though it doesn't look right. It's just a way for Snyk to link back to the building repo, see https://countingup.slack.com/archives/C01RY6B9AG0/p1623580355000500.

But now that I look at that chat log again, I notice that we should include the other label, `io.snyk.containers.image.dockerfile="/Dockerfile-expocli"` since it's a non-default Dockerfile.